packages/manager/apps/pci-kubernetes/src/pages/new/steps/PlanStep.component.tsx

```diff
@@ -10,12 +9,15 @@ import {
   ODS_THEME_TYPOGRAPHY_SIZE,
 } from '@ovhcloud/ods-common-theming';
 import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_TYPE,
   ODS_MESSAGE_TYPE,
   ODS_SPINNER_SIZE,
   ODS_TEXT_LEVEL,
   ODS_TEXT_SIZE,
 } from '@ovhcloud/ods-components';
-import { OsdsMessage, OsdsSpinner, OsdsText } from '@ovhcloud/ods-components/react';
+import { OsdsButton, OsdsMessage, OsdsSpinner, OsdsText } from '@ovhcloud/ods-components/react';
```

Comment:
suggestion: you should migrate thoses components to ODS19 (since it's the pr purpose)

Reply:
Ok
But you added OdsButton, why not add it in ODS19 instead ? Expecially since you already have Badge and Icon in ODS19